.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Tag Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version to tag'
+        required: true
+        type: string
+jobs:
+  create-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Update Files
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |-
+          echo $VERSION > version
+          sed -i "s/v.in-dev/v.${VERSION}/" < docker/dashboard/index.html > docker/dashboard/index.html
+      - name: "Create Tag"
+        env:
+          VERSION: ${{ inputs.version }}
+        run: |-
```

Comment:
You should also configure git author
https://github.com/actions/checkout#push-a-commit-using-the-built-in-token